packages/terra-base/tests/nightwatch/ArrayBase.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
```

Comment:
nit:  have an extra line here between imports don't really need, also could remove an extra line after the imports as well

Reply:
Removed in this commit: https://github.com/cerner/terra-core/pull/1012/commits/de92c42b10d391c997e2d2edd3ee4b606e679213